dashboard/config/levels/custom/music/musiclab_intro_trigger.level

```diff
@@ -0,0 +1,66 @@
+<Music>
+  <config><![CDATA[{
+  "published": true,
+  "game_id": 70,
+  "created_at": "2024-01-09T00:48:10.000Z",
+  "level_num": "custom",
+  "user_id": 976,
+  "properties": {
+    "encrypted": "false",
+    "background": "music-black",
+    "level_data": {
+      "library": "will",
+      "startSources": {
+        "blocks": {
+          "languageVersion": 0,
+          "blocks": [
+            {
+              "type": "when_run_simple2",
+              "id": "when-run-block",
+              "x": 30,
+              "y": 30
+            },
+            {
+              "type": "triggered_at_simple2",
+              "x": 230,
+              "y": 30,
+              "fields": {
+                "trigger": "<field name=\"trigger\">trigger1</field>",
+                "trigger_start": "<field name=\"trigger_start\">next_measure</field>"
+              }
+            }
+          ]
+        }
+      },
+      "toolbox": {
+        "blocks": {
+          "Play": [
+            "play_sound_at_current_location_simple2"
+          ]
+        },
+        "type": "flyout"
+      },
+      "text": "#### Play live!\n\nDid you know you can do a live performance?\n\nAttach a **play** block to the **trigger** block.\n\nPress **Run** and press **1** to trigger the sound.",
+      "validations": [
+        {
+          "conditions": [
+            {
+              "name": "played_sounds_together",
```

Comment:
We have a triggered condition we can use here right?

Reply:
Good memory!  Updated.